src/accordion/Accordion.tsx

```diff
@@ -0,0 +1,29 @@
+import "./_accordion.scss";
+
+import React from "react";
+import classNames from "classnames";
+
+import AccordionItem from "./item/AccordionItem";
+import {AccordionContextProvider} from "./AccordionProvider";
+
+export type AccordionProps = {
+  allowMultipleExpanded?: boolean;
+  allowZeroExpanded?: boolean;
+  defaultExpanded?: string[];
+  customClassName?: string;
+  children: React.ReactNode;
+};
+
+function Accordion(props: AccordionProps) {
+  const {children, customClassName, ...rest} = props;
+
+  return (
+    <AccordionContextProvider {...rest}>
```

Comment:
I don't think using `...rest`, that I mentioned, is not proper for context. We can use `...rest` for `HTMLElement`. 

For example;
You will create your own `input` component and this component will have more features than `HTMLInputElement`. You might need other props but probably you will use core `HTMLInputElement` props. In this case, you can pass core props with `...rest` because you don't need any changes on these props.

In here;
`AccordionContextProvider` not an element or component, that's why we shouldn't use `...rest` here.

Reply:
Thanks for the detailed explanation.